content/docs/traffic-management/route-delegation/header-query.md

```diff
@@ -126,7 +126,7 @@ The following image illustrates the route delegation hierarchy:
    EOF
    ```
 
-3. Create the `child-team2` HTTPRoute resource in the `team2` namespace that matches traffic on the `/anything/team2/bar` exact prefix if the `headerX=valX` request header and `queryX=valX` query parameter are present in the request. Requests that meets these conditions are forwarded to the httpbin app in the `team2` namespace. 
+3. Create the `child-team2` HTTPRoute resource in the `team2` namespace that matches traffic on the `/anything/team2/bar` exact path if the `header2=val2` request header and `query2=val2` query parameter are present in the request. Requests that meet these conditions are forwarded to the httpbin app in the `team2` namespace.
```

Comment:
I believe this should still say `headerX=valX` and `queryX=valX`

Reply:
ah whoops i forgot to revert